test/common/http/http2/http2_frame.h

```diff
@@ -116,6 +116,7 @@ class Http2Frame {
    * @return an Http2Frame that is comprised of the given contents.
    */
   static Http2Frame makeGenericFrame(absl::string_view contents);
+  static Http2Frame makeGenericFrameFromHexDump(absl::string_view contents);
```

Comment:
@adisuissa was the the original intention for `makeGenericFrame`?

Reply:
The intent was that in addition to the known frames, to let the H2 integration fuzzer add some generic data (probably malformed) as an H2 frame (following this [comment](https://github.com/envoyproxy/envoy/pull/10321#discussion_r398566969)).

Reply:
Gotcha -- sorry my question was more if the bytes were hex encoded or not. But thinking back on it, probably it would fail parsing too often if inputs were random bytes and not hex bytes

Reply:
Agree, the fuzzer will have a harder time creating hex strings